### ISSUE TITLE
test warnings: empty slice and true_divide

### ISSUE THREAD

**Issue by jeff-regier** [YosefLab/scVI#327]
```
tests/test_notebook.py::test_notebooks_reproducibility
  /home/travis/build/YosefLab/scVI/.eggs/numpy-1.14.6-py3.7-linux-x86_64.egg/numpy/core/fromnumeric.py:2957: RuntimeWarning: Mean of empty slice.
    out=out, **kwargs)
  /home/travis/build/YosefLab/scVI/.eggs/numpy-1.14.6-py3.7-linux-x86_64.egg/numpy/core/_methods.py:80: RuntimeWarning: invalid value encountered in true_divide
    ret = ret.dtype.type(ret / rcount)
```

**Comment by PierreBoyeau** [comment]
I tried my best to launch the notebook in the configuration it is during testing. I suppose the folder `tests/data` is used to import mini versions of the dataset but I do not manage to open the datasets that way ...

**Comment by PierreBoyeau** [comment]
Hey Jeff,

The first time this warning appears is in the third cell of part "Figure 14. imputation".
The problem is that the test version of `brain_large` is extremely small (4 cells, 8 genes, 4 non null values).
Since the corruption changes 10% of *non zeros* values, the corrupted dataset is the same as the original dataset, which causes indirectly the error.

2 paths can be chosen:
- Make a bigger test `brain_large`
- Change the behavior of this [method](https://github.com/YosefLab/scVI/blob/master/scvi/dataset/dataset.py#L91]), but this looks a like an overkill.

What do you think?

**Comment by jeff-regier** [comment]
Having a larger test dataset seems good. Still under 50 KB ideally. Thanks for working on this issue.

**Comment by jeff-regier** [comment]
Closed by #340
